src/test/java/com/google/step/youtube/YoutubeInfoScraperTest.java

```diff
@@ -64,34 +72,34 @@ public void setUp() {
   @Test
   public void scrapeChannelUploadPlaylist_nonExistentChannelId() throws Exception{
       ChannelListResponse mockResponse = new ChannelListResponse();
-      when(apiResponse.getYoutubeChannelListResponse("CHANNELID_THAT_DOES_NOT_EXIST")).thenReturn(mockResponse.setItems(null));
-      Optional<String> actual = scraper.scrapeChannelUploadPlaylist(apiResponse, "CHANNELID_THAT_DOES_NOT_EXIST");
-      Assert.assertEquals(false, actual.isPresent());
+      when(apiResponse.getYoutubeChannelListResponse(CHANNEL_ID_THAT_DOES_NOT_EXIST)).thenReturn(mockResponse.setItems(null));
+      Optional<String> actual = scraper.scrapeChannelUploadPlaylist(apiResponse, CHANNEL_ID_THAT_DOES_NOT_EXIST);
+      assertEquals(false, actual.isPresent());
   }
 
  @Test
   public void scrapeChannelUploadPlaylist_emptyList() throws Exception{
       ChannelListResponse mockResponse = new ChannelListResponse();
-      when(apiResponse.getYoutubeChannelListResponse("CHANNELID")).thenReturn(mockResponse.setItems(Arrays.asList()));
-      Optional<String> actual = scraper.scrapeChannelUploadPlaylist(apiResponse, "CHANNELID");
-      Assert.assertEquals(false, actual.isPresent());
+      when(apiResponse.getYoutubeChannelListResponse(CHANNEL_ID)).thenReturn(mockResponse.setItems(Arrays.asList()));
+      Optional<String> actual = scraper.scrapeChannelUploadPlaylist(apiResponse, CHANNEL_ID);
+      assertEquals(false, actual.isPresent());
   }
 
   @Test
   public void scrapeChannelUploadPlaylist_channelExists() throws Exception{
```

Comment:
Prefer throwing the most specific exception.

Reply:
Done. Added IOException.